src/screens/styles.ts

```diff
@@ -4,7 +4,89 @@ export const styles = StyleSheet.create({
   container: {
     flex: 1,
     flexDirection: 'column',
-    alignItems: 'center',
+    // alignItems: 'center',
     backgroundColor: 'white',
+    paddingHorizontal: 15, // should use paddingHOrizontal to ensure it stretches across the screen so bg color doesn't change
+    gap: 8,
+    fontFamily: 'Poppins_400Regular',
+  },
+  page: {
+    flex: 1,
+    margin: '20%',
+  },
+  image: {
+    //flex: 1,
+    justifyContent: 'center',
+    width: '100%',
+    height: 231,
```

Comment:
Specifying height in pixels here could make the image look weird on different size screens. Its better to use `aspectRatio: 16 / 9`.